Vizzuality/heco-invest
# Update the login flow to account for 2 fa

This PR adds the 2fa to sign in flow:

Adds the need of using 2fa to sign-in page
Creates the verification code page

## Testing instructions

1. Go to sign-in
2. Fill the form with any user with enabled 2fa ( [barbara.chaves+1@vizzuality.com] / 1234qwerASDF )  
3. You should be redirected to sign-in/code and receive an email with the authentication code
4. Fill the input with the code and confirm

- The inputs should accept only numbers and they are required 
- If the code is wrong, an error message will be shown

5. You should be redirected to the next screen (the example user is a light user, so you go to the accounts screen)

Try to login with a user without 2fa to make sure the flow wasn't affected

## Tracking

[1136](https://vizzuality.atlassian.net/browse/LET-1136)
